### PR TITLE
Add new user to the active sprint

### PR DESCRIPTION
**Pull request by Sekru**
Added babel compiler from es7 to node 6.11

### CODE REVIEW

File: firebase job/functions/es7.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        user: 'fruits.future.processing@gmail.com',
+        pass: 'Admin!23'
+    }
+});
+
+const emailReminder = user => {
+    const mailOptions = {
+        from: 'fruits.future.processing@gmail.com',
+        subject: 'Fruits reminder',
+        text: 'W poniedziałek jest Twoja kolej aby przynieść owoce. Jeśli nie możesz tego zrobic, zorganizuj zastępstwo.',
```
Comment (carl-st) [inline firebase job/functions/es7.js]: We should decide in what language we have the app. Title is in english and text in polish. UI is in english  :stuck_out_tongue:

Comment (Sekru) [reply firebase job/functions/es7.js]: It's just email content

Comment (carl-st) [reply firebase job/functions/es7.js]: I meant subject, not title

Comment (carl-st) [reply firebase job/functions/es7.js]: Okay, we'll do polish translation in another PR :smiley:

Comment (Sekru) [reply firebase job/functions/es7.js]: aaaaaaa OK I will change title to PL

**Pull request merged**